soc/ite/ec/it8xxx2/Kconfig.soc

```diff
@@ -1,9 +1,20 @@
 # Copyright (c) 2020 ITE Corporation. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+config SOC_SERIES_ITE_IT8XXX2
+	bool "ITE IT8XXX2 implementation"
+	#depends on RISCV
+	# RV32IAFC is an uncommon configuration which is not supported by
+	# default in most toolchains, causing link-time errors.
+	select CPU_HAS_FPU if "$(ZEPHYR_TOOLCHAIN_VARIANT)" != "zephyr" || RISCV_ISA_EXT_M
+	select SOC_FAMILY_ITE_EC
+	select HAS_PM
+	help
+	    Enable support for ITE IT8XXX2
+
 choice
-prompt "ITE IT8XXX2 system implementation"
-depends on SOC_SERIES_ITE_IT8XXX2
+	prompt "ITE IT8XXX2 system implementation"
+	depends on SOC_SERIES_ITE_IT8XXX2
```

Comment:
Prompts vanish and the below move to `Kconfg` as per above, the part that says in this file is minimal, basically:
```
config SOC_IT8XXX2
    bool
    select SOC_SERIES_ITE_IT8XXX2
```